vcbuild.bat

```diff
@@ -577,6 +577,7 @@ set TAG=
 set FULLVERSION=
 :: Call as subroutine for validation of python
 call :run-python tools\getnodeversion.py > nul
+if errorlevel 1 exit /b 1
```

Comment:
This is ugly. Basically, we run `tools\getnodeversion.py`, ignore the output, exit if it failed, then run it again, this time directly using a variable that's supposed to be internal to `find_python.cmd`.

Reply:
could change the fake call to `call :run-python --version`

Reply:
It would still be a hacky workaround for the limitation of `run-python`.

Reply:
No argument here. This file has a lot of `cmd`ism hacks, mainly because you can't redirect stdout into a variable.
I'll check if there's a way to use `call` as the `for` command.

Reply:
I've checked before, there isn't. So a subroutine doesn't work as a replacement for the `python` command.

An alternative would be to replace `run-python` with `find-python` that would just populate a variable to be used instead of `python`. But then one would have to make sure that `find-python` is run before the first usage of Python and that it's not run if Python isn't needed, whenever a significant change is introduced into `vcbuild.bat`.

Honestly, I'd prefer to just revert 614dbbd728bba2ed05c3861a2a7316c399d835e9 since it introduces a lot of complexity in order to support an edge case that has never affected anyone to my knowledge. IMO it was landed prematurely (only one approval and not thoroughly tested).